koku/reporting/migrations/0261_ocpgcpcostlineitemprojectdailysummaryp_pod_credit.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-10-17 15:41
```

Comment:
We should move these postgres migrations to their own pull request.

Reply:
I _Can_ do this but is it strictly necessary especially with this additive case, and how we run migrations. Sure in stage the migration will run while the app is redeploying but that happens regardless now. As for Prod we can push out the migration separately from this code being released in app-interface anyway.

Reply:
https://github.com/project-koku/koku/pull/3949